pages/glossary/essential-text-change.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Essential Text Change
+key: essential-text-change
+unambiguous: true
+objective: false
+input_aspects:
+  - Accessibility tree
+  - CSS styling
+  - DOM tree
+---
+
+An _essential text change_ occurs when the modification of the text content of an HTML element is necessary to convey accurate and timely information to the user. An _essential text change_ may appear in any of the following forms:
```

Comment:
Would something like a clock or a count-down be considered essential. For example on a new years calendar, an auctioning site counting down when the auction closes, or a ticket site counting down to when tickets go on sale?

Reply:
I think if it is a core functionality of the website then it would be considered essential. E.g., in a web page that is only a large clock counting down to new year then the text change of the clock would be essential.